docker-images/kafka/kafka-thirdparty-libs/2.3.x/pom.xml

```diff
@@ -15,6 +15,13 @@
         </license>
     </licenses>
 
+    <repositories>
+      <repository>
+        <id>jcenter</id>
+          <url>https://jcenter.bintray.com/</url>
+      </repository>
+    </repositories>
```

Comment:
This is where CC is published?

Reply:
Yes

Reply:
Ah sorry, this isn't the right place, I am looking at the wrong file. I'll remove it